website/views.py

```diff
@@ -57,65 +63,74 @@ def get_question(request, question_id=None, pretty_url=None):
+    thisuserdownvote = question.userDownVotes\
+                               .filter(id=request.user.id).count()
     net_count = question.userUpVotes.count() - question.userDownVotes.count()
-    
+
     ans_votes = []
 
     for vote in answers:
-        net_ans_count  = vote.userUpVotes.count() - vote.userDownVotes.count()
-        ans_votes.append([vote.userUpVotes.filter(id=request.user.id).count(),vote.userDownVotes.filter(id=request.user.id).count(),net_ans_count])
-    #for (f,b) in zip(foo, bar):
-    main_list = zip(answers,ans_votes)
+        net_ans_count = vote.userUpVotes.count() - vote.userDownVotes.count()
+        ans_votes.append([vote.userUpVotes.filter(id=request.user.id).count(),
+                          vote.userDownVotes.filter(id=request.user.id).count()
+                          , net_ans_count])
+    # for (f,b) in zip(foo, bar):
+    main_list = zip(answers, ans_votes)
     context = {
         'ans_count': ans_count,
         'question': question,
-        'sub_category':sub_category,
+        'sub_category': sub_category,
         'main_list': main_list,
         'form': form,
         'thisUserUpvote': thisuserupvote,
         'thisUserDownvote': thisuserdownvote,
         'net_count': net_count,
-        'num_votes':question.num_votes,
-        'ans_votes':ans_votes
+        'num_votes': question.num_votes,
+        'ans_votes': ans_votes,
+
     }
     context.update(csrf(request))
     # updating views count
     question.views += 1
     question.save()
-   
+
     return render(request, 'website/templates/get-question.html', context)
-    
-# post answer to a question, send notification to the user, whose question is answered
+
+
+# post answer to a question, send notification to the user,
+# whose question is answered
 # if anwser is posted by the owner of the question, no notification is sent
 @login_required
-def question_answer(request,qid):
-   
+def question_answer(request, qid):
+
     context = {}
     question = get_object_or_404(Question, id=qid)
-   
+
     if request.method == 'POST':
         form = AnswerQuestionForm(request.POST)
         answers = question.answer_set.all()
         answer = Answer()
-        
+
         answer.uid = request.user.id
         if form.is_valid():
             cleaned_data = form.cleaned_data
             qid = cleaned_data['question']
             body = str(cleaned_data['body'])
-            #print body
+            # print body
             # body = body.replace("\\r", '')
             # body = body.replace("\\n", '')
             # body = body.replace("\\t", '')
-            answer.body = body.splitlines()    
+            answer.body = body.splitlines()
             answer.question = question
-            answer.body = body.encode('unicode_internal')     
+            answer.body = body.encode('unicode_internal')
             answer.save()
             # if user_id of question not matches to user_id of answer that
```

Comment:
Commented code is unhealthy unless really necessary. Please verify and remove if not needed.